src/fiskaltrust.Middleware.Interface.Client.Http/fiskaltrust.Middleware.Interface.Client.Http.csproj

```diff
@@ -5,11 +5,19 @@
     <LangVersion>8</LangVersion>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetFramework)' != 'net40' AND '$(TargetFramework)' != 'net461'">
+    <DefineConstants>SYSTEM_TEXT_JSON</DefineConstants>
+  </PropertyGroup>
+
   <ItemGroup>
     <PackageReference Include="fiskaltrust.interface" Version="1.3.73-rc5-25197-83747" />
     <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' != 'net40' AND '$(TargetFramework)' != 'net461'">
```

Comment:
I think a similar discussion as for other things, but should we scope this to != netstandard also?

Reply:
since the version of System.Text.Json supplied with the frameworks behaves a bit differently depending on the framework version I don't think that's a good idea. When trying this previously it broke the de/serialization in many ways ^^